Makefile

```
@@ -25,11 +25,19 @@ export SUBCTL_VERSION
 
 # Targets to make
 
-e2e: E2E_ARGS=cluster1 cluster2
+package-charts: submariner-k8s-broker-$(CHARTS_VERSION).tgz submariner-operator-$(CHARTS_VERSION).tgz
+
+local-helm-repo: package-charts
+	mkdir $(HELM_REPO_LOCATION)
+	tar -zxf submariner-k8s-broker-$(CHARTS_VERSION).tgz -C $(HELM_REPO_LOCATION)
+	tar -zxf submariner-operator-$(CHARTS_VERSION).tgz -C $(HELM_REPO_LOCATION)
```

Avoid relying on a specific set of packages:

```suggestion
	for archive in $^; do \
	    tar xzf $$archive -C $(HELM_REPO_LOCATION) \
	done
```